network-api/networkapi/templates/donate/fragments/formassembly_body.html

```diff
@@ -74,7 +69,7 @@
         </script></div>
         <form method="post" action="https://mozillafoundation.tfaforms.net/api_v2/workflow/processor" class=" labelsAbove" id="12" role="form" enctype="multipart/form-data" style="padding:0;">
             <div class="oneField field-container-D  labelsAbove  " id="tfa_95-D">
-                <label id="tfa_95-L" class="label preField" for="tfa_95">I need</label><br><div class="inputWrapper"><select aria-required="true" id="tfa_95" name="tfa_95" title="I need" class="required"><option value="">Please select...</option>
+                <label id="tfa_95-L" class="label preField" for="tfa_95">{% trans "I need" %}</label><br><div class="inputWrapper"><select aria-required="true" id="tfa_95" name="tfa_95" title="{% trans "I need" %}" class="required"><option value="">{% trans "Please select..." %}</option>
                     <option value="tfa_194" id="tfa_194" data-conditionals="#tfa_212,#submit_button,#tfa_214,#tfa_163" class="">Help with the donation form</option>
```

Comment:
It would be great if we could show localized text for these dropdown options. Let's wrap them in `trans`

Reply:
Thanks for the feedback Mavis! This makes sense to me, I did not know that dropdown options can also be wrapped in `trans`! I have updated the PR, and re-requested review for whenever you have time 👍